HIP-0013.md

```diff
@@ -0,0 +1,130 @@
+
+### DNS: A Strategy for Adoption
+This specification is fully compatible with ICANN domains, which dramatically broadens the potential user base. By growing the identity use-case for DNS domains, we can position Handshake as the superior alternative – private, decentralized, and censor-resistant. By building tools and services around IDNS that support both HNS and DNS domains, we can reach a much wider audience beyond the crypto community.
+
+## TXT Specification
+
+IDNS TXT records begin with **`IDNS1`** (version 1), and use the following format containing a record type, key, and value: 
+
+<pre><code><b>IDNS1 &lt;type&gt;:&lt;key&gt;=&lt;value&gt;</b>
+</code></pre>
+
+IDNS TXT records can contain one or more records in key=value pairs, separated by spaces:
+<pre><code><b>IDNS1</b> profile:avatar=<ins>http://pic.com/me.png</ins> service:com.twitter=<ins>aoxborrow</ins>
+</code></pre>
+
+Records that contain spaces must be single quoted. Single quotes in the record can be escaped with a backslash:
+<pre><code><b>IDNS1</b> profile:location=<ins>'Berkeley, CA'</ins> profile:bio=<ins>'I\'m a passionate advocate for domain innovation.'</ins></code></pre>
+
+### Profile Records (`profile`)
+Human-readable user profile text data. 
+  - **`<key>`** The profile record name, any text key is valid, no spaces.
+  - **`<value>`** The profile record data.
+<pre><code><b>IDNS1 profile:&lt;key&gt;=&lt;value&gt;</b>
+
+<i>Examples:</i>
+IDNS1 profile:name=aox
+IDNS1 profile:avatar=https://hns.id/domain/aox.hns/avatar.png
+IDNS1 profile:location='Berkeley, CA'
+IDNS1 profile:email=hey@me.com
+</code></pre>
+
+### Service Records (`service`)
+Service keys must be made up of a reverse dot notation for a namespace which the service owns, for example, DNS names (e.g. .com, .io) or HNS name (i.e. .hns). Service Keys must contain at least one dot. See [EIP-634](https://eips.ethereum.org/EIPS/eip-634#service-keys).
+  - **`<key>`** A service key associated with the user, e.g. `com.github`.
+  - **`<identifier>`** Identifier for the service, e.g. Github username. 
+<pre><code><b>IDNS1 service:&lt;key&gt;=&lt;identifier&gt;</b>
+
+<i>Examples:</i>
+IDNS1 service:com.twitter=aoxborrow
+IDNS1 service:com.github=0xstefan
+IDNS1 service:io.keybase=pinheadmz
+</code></pre>
+
+### Wallet Records (`wallet`)
+Cryptocurrency wallet addresses. Uses [SLIP44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) to specify different blockchains.
+  - **`<cointype>`** The type of cryptocurrency, using SLIP44 cointype symbol or number, e.g. `hns` or `5353`
+  - **`<address>`** Wallet address for the specific cryptocurrency.
+<pre><code><b>IDNS1 wallet:&lt;cointype&gt;=&lt;address&gt;</b>
+
+<i>Examples:</i>
+IDNS1 wallet:btc=1Dvf5aQkVrVJeasRUDiLATGWLBGFWA9ovY
+IDNS1 wallet:hns=hs1qshuyulxra3pqpwr40303t8pn79232zztuk4qgz
+IDNS1 wallet:5353=hs1qshuyulxra3pqpwr40303t8pn79232zztuk4qgz
+IDNS1 wallet:eth=0xa08abA528da538cd3547a73C93d51B90201294B6
```

Comment:
could put that record type/key is not case sensitive. i wouldn't want to specify either way for addresses since some of them may be case-sensitive and don't really know what it would say. a client app should just parse it as-is

Reply:
I lowercased all the addresses, but i don't think there's anything to say about preserving case

Reply:
Oh sorry maybe I should've been more clear. Making addresses lowercase will _break_ a few ones like BTC and ETH. They require their caps to stay caps.

>i wouldn't want to specify either way for addresses since some of them may be case-sensitive and don't really know what it would say. a client app should just parse it as-is

Yeah I think this is fine. Anyway TXT records themselves are case-preserving so it's up to the user/their app to make sure the addresses have the right caps.